Misc/NEWS.d/next/Library/2019-12-07-18-58-44.bpo-38994.IJYhz_.rst

```diff
@@ -0,0 +1 @@
+Implement ``__class_getitem__`` for ``os.PathLike``, ``pathlib.Path``
```

Comment:
This news item is slightly misleading as the change was made to `pathlib.PurePath`, not `pathlib.Path`.

Reply:
And @isidentical forgot to give himself credit for the change. :)